README.md

```diff
@@ -34,6 +34,85 @@ ERROR: SystemError: opening file foo: No such file or directory
  in open at iostream.jl:99
 ```
 
+### @mendable
+
+Some functions you want to mock are called from the function you are testing. If the function you are testing is compiled, you will not be able to mock the internal function. That is where `@mendable` can help, it will allow you to mock the internal functions to your needs. As show in this example.
+
+```
```

Comment:
Should use syntax highlighting on code blocks:
```
```julia
julia> println("helloworld!")
``` ...